deploy/roles/postgresql-server-config/tasks/main.yml

```diff
@@ -16,13 +16,16 @@
   become_method: sudo
   lineinfile:
     dest: "{{ pp_datadir }}/pg_hba.conf"
-    regexp: '^(.*)::1/128(.*)md5'
+    regexp: '^host(.*)postgres(.*)md5'
     backrefs: yes
-    line: '\1::1/128\2trust'
+    line: 'host\1postgres\2trust'
```

Comment:
Will these work locally?

Reply:
Added a line to the defaults for pg_hba.conf to make this work in Vagrant.